index.html

```diff
@@ -11580,7 +11580,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<p>Authors MAY, with caution, use aria-hidden to hide visibly rendered content from assistive technologies <em>only</em> if the act of hiding this content is intended to improve the experience for users of assistive technologies by removing redundant or extraneous content. Authors using aria-hidden to hide visible content from screen readers MUST ensure that identical or equivalent meaning and functionality is exposed to assistive technologies.</p>
 				<p class="note">Authors are advised to use extreme caution and consider a wide range of disabilities when hiding visibly rendered content from assistive technologies. For example, a sighted, dexterity-impaired individual may use voice-controlled assistive technologies to access a visual interface. If an author hides visible link text "Go to checkout" and exposes similar, yet non-identical link text "Check out now" to the accessibility API, the user may be unable to access the interface they perceive using voice control. Similar problems may also arise for screen reader users. For example, a sighted telephone support technician may attempt to have the blind screen reader user click the "Go to checkout" link, which they may be unable to find using a type-ahead item search ("Go to…").</p>
 				<p class="note">At the time of this writing, <code><sref>aria-hidden</sref>="false"</code> is known to work inconsistently in browsers. As future implementations improve, use caution and test thoroughly before relying on this approach.</p>
-				<p class="note">It is recommended that authors key visibility of elements off this attribute, rather than change visibility and separately update this <a>property</a>. <abbr title="Cascading Style Sheets">CSS</abbr> 2 introduced a way to <cite><a href="https://www.w3.org/TR/css3-selectors/#attribute-selectors">select on attribute values</a></cite> ([[CSS3-SELECTORS]]). The following <abbr title="Cascading Style Sheets">CSS</abbr> declaration makes content visible unless the <sref>aria-hidden</sref> attribute is <code>true</code>; scripts need only update the <span>value</span> of this attribute to change visibility:</p>
+				<p class="note">It is recommended that authors key visibility of elements off this attribute, rather than change visibility and separately update this <a>property</a>. <abbr title="Cascading Style Sheets">CSS</abbr> 2 introduced a way to <cite><a href="https://www.w3.org/TR/css4-selectors/#attribute-selectors">select on attribute values</a></cite> ([[CSS4-SELECTORS]]). The following <abbr title="Cascading Style Sheets">CSS</abbr> declaration makes content visible unless the <sref>aria-hidden</sref> attribute is <code>true</code>; scripts need only update the <span>value</span> of this attribute to change visibility:</p>
```

Comment:
```suggestion
				<p class="note">It is recommended that authors key visibility of elements off this attribute, rather than change visibility and separately update this <a>property</a>. <abbr title="Cascading Style Sheets">CSS</abbr> 2 introduced a way to <cite><a href="https://www.w3.org/TR/css3-selectors/#attribute-selectors">select on attribute values</a></cite> ([[CSS3-SELECTORS]]). The following <abbr title="Cascading Style Sheets">CSS</abbr> declaration makes content visible unless the <sref>aria-hidden</sref> attribute is <code>true</code>; scripts need only update the <span>value</span> of this attribute to change visibility:</p>
```